src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -398,8 +398,7 @@ trait Definitions extends api.StandardDefinitions {
 
     lazy val TypeConstraintClass   = requiredClass[scala.annotation.TypeConstraint]
     lazy val SingletonClass        = enterNewClass(ScalaPackageClass, tpnme.Singleton, AnyTpe :: Nil, ABSTRACT | TRAIT | FINAL) markAllCompleted
-    lazy val SerializableClass     = JavaSerializableClass
-    lazy val JavaSerializableClass = requiredClass[java.io.Serializable] modifyInfo fixupAsAnyTrait
+    lazy val SerializableClass     = requiredClass[java.io.Serializable] modifyInfo fixupAsAnyTrait
```

Comment:
Failure at Definitions.hijackedCoreClasses below.

Between the analytic and the synthetic lies the mysterious outlaw zone of... the hijacked!